cl/favorites/tests.py

```diff
@@ -924,7 +924,7 @@ async def test_prayers_integration(self) -> None:
             email_text_content,
         )
         self.assertIn(
-            f"You requested it on {prayer_1.date_created.strftime("%b %d, %Y")}",
+            f"You requested it on {prayer_1.date_created.strftime("%b %-d, %Y")}",
```

Comment:
We saw this test failing too. I think you should delete this commit if you can, and rebase on main. That should get you the fix.